src/log.js

```diff
@@ -516,17 +546,11 @@ class Log extends GSet {
    * @returns {Promise<Log>}
    */
   static async fromMultihash (ipfs, identity, hash,
-    { access, length = -1, exclude, onProgressCallback, sortFn, timeout, format } = {}) {
+    { access, length = -1, exclude = [], concurrency, sortFn, onProgressCallback } = {}) {
```

Comment:
@haadcode any reason we removed the timeout?

Reply:
Ah, I misread the original code and thought timeout functionality was removedfrom entry-io.js. Will add it back 👍

Reply:
This has been addressed now by adding back the timeout and adding tests for the timeout to work when fetching entries.